datafusion/common/src/lib.rs

```diff
@@ -83,6 +83,9 @@ pub use functional_dependencies::{
 use hashbrown::hash_map::DefaultHashBuilder;
 pub use join_type::{JoinConstraint, JoinSide, JoinType};
 pub use null_equality::NullEquality;
+#[cfg(feature = "object_store")]
+/// Reexport object_store crate
+pub use object_store;
```

Comment:
it's exported publicly elsewhere as `datafusion::object_store`
let's not export it from the common crate